tools/clang/test/DXC/private_data.test

```diff
@@ -0,0 +1,26 @@
+// Set private data
+// RUN: echo private data > %t.private.txt
+// RUN: %dxc %S/Inputs/smoke.hlsl /D "semantic = SV_Position" /T vs_6_0 /Zi /Qembed_debug /DDX12 /Fo %t.private.smoke.cso
+
+// RUN: %dxc %t.private.smoke.cso /dumpbin /setprivate %t.private.txt /Fo %t.private.cso
+
+// RUN: %dxc %t.private.cso /dumpbin /Qstrip_priv /Fo %t.noprivate.cso
+
+// RUN: %dxc %t.private.cso /dumpbin /getprivate %t.private1.txt
+// RUN: FileCheck %s --input-file=%t.private1.txt --check-prefix=PRIVATE1
+// PRIVATE1:private data
+
+// Appending and removing private data blob roundtrip
+// RUN: fc %t.private.smoke.cso %t.noprivate.cso
```

Comment:
The Windows FC command and the Unix FC command do _very_ different things. This line is almost certainly not doing what you think it is.